### ISSUE TITLE
Missing trailing slash

### ISSUE THREAD

**Issue by HernanZh** [lwsjs/local-web-server#56]
Hello, I like using this package as a simple server inside my projects folder and as such I always use the server's filetree to navigate to the project I like to load in the browser. However, I recently updated and now there is always a missing slash at the end.

For example when I click on "project1" folder, it loads "localhost:8000/project1" instead of "localhost:8000/project1/". This results in 404 errors for all javascript and css files.

**Comment by 75lb** [comment]
i've come across this issue before, it's an issue in the underlying module supplying the directory-browsing feature.. will look into it, thanks.

**Comment by HernanZh** [comment]
Oops, I forgot about this issue. Either way, thanks for this useful tool. I hope you figure out this issue.

**Comment by 75lb** [comment]
this is resolved in v2 which is in progress and [available for preview](https://github.com/lwsjs/local-web-server/tree/next).. any feedback welcome.

**Comment by HernanZh** [comment]
Hm, annoyingly enough, one of the node modules is giving me an error, requiring node v7.6+
--> https://github.com/koajs/bodyparser/issues/66

I'll have to check this later

**Comment by 75lb** [comment]
yeah, sorry about that - koa is node v7.6 at the minimum these days (for async/await support).

**Comment by HernanZh** [comment]
Alright, I installed the latest node. Seems to work! 
Forcing the new node version on other users could be problematic though, but that's up to you...

**Comment by 75lb** [comment]
glad it works! I didn't choose to force the latest node on users - this constraint was forced upon by the Koa project unfortunately. I don't agree with it but don't have much choice.

**Comment by 75lb** [comment]
fixed and released in v2.0.0

**Comment by sergebat** [comment]
I'd like to report that this issue reappeared on the version 3.0 (reproduced on Win 10, 3.0.4). 2.0 works fine for me. 

Fragment of listing html that I see:
```
    <div id="wrapper">
      <h1><a href="http://127.0.0.1:8000/">~</a> / </h1>
      <ul id="files" class="view-tiles"><li><a href="http://127.0.0.1:8000/node_modules" class="icon icon-directory" title="node_modules"><span class="name">node_modules</span><span class="size"></span><span class="date">2019-7-15 2:59:38 PM</span></a></li>
<li><a href="http://127.0.0.1:8000/test" class="icon icon-directory" title="test"><span class="name">test</span><span class="size"></span><span class="date">2019-7-15 3:00:32 PM</span></a></li>
<li><a href="http://127.0.0.1:8000/package-lock.json" class="icon icon icon-json icon-application-json" title="package-lock.json"><span class="name">package-lock.json</span><span class="size">56802</span><span class="date">2019-7-15 2:59:38 PM</span></a></li></ul>
    </div>
```

**Comment by 75lb** [comment]
Hmm, i can't reproduce this on macOS and in it theory it should not behave any differently on Windows.. I will try and reproduce it on Windows later.

**Comment by 75lb** [comment]
I can't reproduce on Windows either. I installed local-web-server v3.0.4 on Windows 10 (version 1809), created a folder, a sub-folder and some files and could navigate the sub-folders without issue. 

Could you post a specific, failing reproduction case please which I can use to see and debug the issue.

**Comment by sergebat** [comment]
> and could navigate the sub-folders without issue

Let me note that basic navigation indeed works. The problem is that it opens the folder "localhost:8000/project1" instead of "localhost:8000/project1/" as stated in the original issue. It breaks relative urls for assets referenced by index.html in localhost:8000/project1. 

If you still cannot reproduce it, I guess I can indeed commit the repo, but the only thing I did was to:
* mkdir repro
* cd repro
* npm install local-web-server 
* mkdir test
* node_modules\.bin\ws.cmd 

Open 127.0.0.1:8000 and observe no trailing slash for the projects.

**Comment by 75lb** [comment]
> It breaks relative urls for assets referenced by index.html in localhost:8000/project1.

I can't reproduce this, relative links work fine for me. I'm sorry to keep asking but I need a specific reproduction case which i can run and see failing. Please post your index.html code which contains a relative URL link which fails.

Locally, I created an index.html which displays images loaded via relative URLs and everything loads correctly.

**Comment by sergebat** [comment]
Sure:
- https://github.com/sergebat/lws-reproduce-56 
- npm install
- run.cmd 

Opened localhost:8000 in chrome and saved resulting listing as:
https://github.com/sergebat/lws-reproduce-56/blob/06c5a0cd919201855f71e695afbeaf2901dd7e17/listing%20directory%20_.html#L347

Let me try the same on osx/ubuntu a bit later.

**Comment by 75lb** [comment]
Great, thanks for that. Yes, now I see what you mean. It's an issue on mac too.

It looks like I will need to fork and patch `serve-index` again (which was [how I solved this back in 2017](https://github.com/lwsjs/serve-index/commit/5884492b7fc43d6f9b7e52eee5ede2a3490ef1d2)).

Will fix ASAP.

**Comment by 75lb** [comment]
Fixed and released in lws-index v1.0.5. Please reinstall local-web-server to pick up the change. 

If you find any more issues, please let me know. 👍
